rfcs/0185-redistribute-redistributable.md

```diff
@@ -0,0 +1,225 @@
+---
+feature: redistribute-redistributable
+start-date: 2024-12-15
+author: Ekleog
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: https://github.com/NixOS/nixpkgs/issues/83884
+---
+
+# Summary
+[summary]: #summary
+
+Make Hydra build and provide all redistributable software, while making sure installation methods stay as fully free as today.
+
+# Motivation
+[motivation]: #motivation
+
+Currently, Hydra builds only free software and unfree redistributable firmware.
+This means that unfree redistributable software needs to be rebuilt by all the users.
+For example, using MongoDB on a Raspberry Pi 4 (aarch64, which otherwise has access to hydra's cache) takes literally days and huge amounts of swap.
+
+Hydra could provide builds for unfree redistributable software, at minimal added costs.
+This would make life much better for users of such software.
+Especially when the software is still source-available even without being free software, like MongoDB.
```

Comment:
If the intent isn’t to commit to providing any specific package then I think using a concrete example of MongoDB in the motivation section is misleading, as accepting this RFC does not necessarily mean that this motivation will be addressed.